pkg/policy/validate_controller.go

```diff
@@ -525,7 +524,9 @@ func checkAutoGenRules(policy *kyverno.ClusterPolicy) bool {
 	ruleCount := 1
 	for _, rule := range policy.Spec.Rules {
 		if utils.ContainsString(rule.MatchResources.ResourceDescription.Kinds, "Pod") {
-			podRuleName = append(podRuleName, rule.Name)
+			if rule.MatchResources.ResourceDescription.Selector == nil || rule.MatchResources.ResourceDescription.Name == "" {
```

Comment:
Do we also need to check these attributes in `exclude`?

Reply:
yes! adding.